warehouse/templates/packaging/detail.html

```diff
@@ -91,7 +91,12 @@ <h1 class="package-header__name">
       {% else %}
       <a class="status-badge status-badge--good" href="{{ request.route_path('packaging.project', name=release.project.name) }}">Latest Version</a>
       {% endif %}
-      <p class="package-header__date">{{ humanize(release.created) }}</p>
+      <p class="package-header__date">
+        <span class="status-badge">
```

Comment:
I wasn't sure if this was acceptable use of the `status-badge` class but I could also try using the `badge` class.

Reply:
I think it's the appropriate class